src/babel/api/register/node.js

```diff
@@ -160,7 +161,7 @@ hookExtensions(util.canCompile.EXTENSIONS);
 
 export default function (opts = {}) {
   if (opts.only != null) only = util.arrayify(opts.only, util.regexify);
-  if (opts.ignore != null) ignore = util.arrayify(opts.ignore, util.regexify);
+  if (opts.ignore != null) ignore = util.arrayify(opts.ignore, util.regexify, isFunction);
```

Comment:
But what do you gain by having this be a third param instead of composing the function with the normal map function? Conceptually you are still mapping the value, you are just mapping it to its original value.